ipld/blockstore/src/lib.rs

```diff
@@ -71,13 +71,29 @@ pub trait BlockStore: Store {
         self.put_raw(bytes, code)
     }
 
+    /// Put an object in the block store and return the Cid identifier.
```

Comment:
:angry:

Reply:
Once everything has been moved to `anyhow::Error`, we'll no longer need the duplicate functions.

Reply:
I wonder if we're already there, actually.

Reply:
@lemmih There are still some methods in blockstore that are using `Box<dyn StdError>`.

Reply:
it was supposed to be unlimited pto not unlimited conflicts :(